### DISCUSSION TITLE
最近打开主页会白屏什么都不显示的问题

### DISCUSSION THREAD

**Discussion by xfgryujk** [xfgryujk/blivechat#216] [公告]
​最近blivechat用的CDN被uBlock Origin屏蔽了，如果浏览器装了这个插件，打开blivechat主页时可能会白屏什么都不显示。但OBS里的显示不受影响 。通过公共服务器使用没有这个问题，因为我临时改成直接打包，不用CDN了

详情参考 https://github.com/xfgryujk/blivechat/issues/214

**Comment by ITManCHINA** [comment]
> ​最近blivechat用的CDN被uBlock Origin屏蔽了，如果浏览器装了这个插件，打开blivechat主页时可能会白屏什么都不显示。但OBS里的显示不受影响 。通过公共服务器使用没有这个问题，因为我临时改成直接打包，不用CDN了
> 
> 详情参考 #214

所以现在BLC现在打不开也是因为这个原因嘛
![1721199730588](https://github.com/user-attachments/assets/8629920c-e9bf-4484-8cdd-1f999f54b100)

**Reply by xfgryujk** [reply]
这个是端口冲突了
